### ISSUE TITLE
archive 3.3.3, 3.3.4 Breaks Image Package for Flutter Web 

### ISSUE THREAD

**Issue by mdrideout** [brendan-duncan/archive#222]
Flutter Web compile error when using [image](https://pub.dev/packages/image)

```bash
Waiting for connection from debug service on Chrome...
../../../../.pub-cache/hosted/pub.dartlang.org/archive-3.3.3/lib/src/util/aes_decrypt.dart:1:8: Error: Not found: 'dart:ffi'
import 'dart:ffi';
       ^
```

**Temporary Fix:** roll back archive version
```yaml
dependency_overrides:
  archive: '3.3.2'
```

**Comment by brendan-duncan** [comment]
archive 3.3.4 shouldn't have 'dart:ffi' in it

**Comment by mdrideout** [comment]
See comparison here: https://github.com/brendan-duncan/archive/compare/3.3.2...3.3.3#diff-3b68a24de2db74d3f45ed99a6bf6939787bc09718e5624a0c088b4acbd051e17

File: lib/archive.dart

Addition of `export 'src/util/aes_decrypt.dart';`

**Comment by mdrideout** [comment]
> archive 3.3.4 shouldn't have 'dart:ffi' in it

OK my system was not pulling 3.3.4 without using override for some reason. Confirmed fixed in 3.3.4

Thanks!
